main.cpp

```diff
@@ -1,4 +1,20 @@
 #include <iostream>
+//Circular array implementation of a queue
+
+class Queue{
+private:
+    int front_index;
+    int back_index;
+    int items_amt;
```

Comment:
If you don't have more members, you can use shorter names, e.g. front, back, items/count/total